packages/python-pulp-cli-deb/python-pulp-cli-deb.spec

```diff
@@ -0,0 +1,71 @@
+%{?scl:%scl_package python-%{pypi_name}}
+%{!?scl:%global pkg_name %{name}}
```

Comment:
Does this make sense for the `rpm/3.14` branch as well?

Reply:
yes, the `?` makes it optional ;)